modules/playlists/src/main/java/org/opencastproject/playlists/PlaylistService.java

```diff
@@ -41,7 +42,6 @@
 import org.opencastproject.util.NotFoundException;
 import org.opencastproject.util.requests.SortCriterion;
 
-import com.entwinemedia.fn.data.Opt;
```

Comment:
Extra credit for removing Entwine Fn code :tada:

Reply:
I know, right? :heart: